src/components/atoms/ErrorMessage/ErrorMessage.tsx

```diff
@@ -53,6 +54,7 @@ const ErrorMessage = ({ children, className, id, ...rest }: ErrorMessageProps) =
   if (theme.errorMessage.iconVariant) {
     const iconMap = {
       'circle-exclamation': CircleExclamationIcon,
+      'circle-exclamation2': CircleExclamation2Icon,
```

Comment:
maybe we can delete all 3 we don't use for this component and leave just a new one?